docs/storybook/src/components/MessageCenterField.stories.tsx

```diff
@@ -34,4 +41,153 @@ const meta = {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    ["1", "2", "3", "4"].forEach((num) => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Info, `Message ${num}`)
+      );
+    });
+  });
+
+  return (
+    <div>
+      <Story />
+    </div>
+  );
+};
+
+export const Notification: Story = {
+  decorators: NotificationDecorator,
+};
+
+//Error Use Case
+
+const ErrorDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    ["1", "2", "3", "4"].forEach((num) => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Error, `Error ${num}`)
+      );
+    });
+  });
+
+  return (
+    <div>
+      <Story />
+    </div>
+  );
+};
+
+export const Error: Story = {
+  decorators: ErrorDecorator,
+};
+
+//Detailed Message
+
+const DetailedDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    [1, 2, 3, 4].forEach((num) => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(
+          num,
+          "This is the brief message",
+          "This is the detailed message"
+        )
+      );
+    });
+  });
+
+  return (
+    <div>
+      <Story />
+      <Button
+        styleType="cta"
+        onClick={() => {
+          MessageManager.clearMessages();
+          MessageManager.addToMessageCenter(
+            new NotifyMessageDetails(
+              1,
+              "This is the brief message",
+              "This is the detailed message"
+            )
+          );
+        }}
+      >
+        Detailed Message
+      </Button>
+    </div>
+  );
+};
+
+export const Detailed: Story = {
+  decorators: DetailedDecorator,
+};
+
+//Dynamic Use Case
+
+const menuItems = (close) => [
+  <MenuItem
+    onClick={() => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Info, `Info Message`)
+      );
+    }}
+  >
+    Info{" "}
+  </MenuItem>,
+  <MenuItem
+    onClick={() => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Error, `Error Message`)
+      );
+    }}
+  >
+    Error
+  </MenuItem>,
+  <MenuItem
+    onClick={() => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(
+          1,
+          "This is the brief message",
+          "This is the detailed message"
+        )
+      );
+    }}
+  >
+    Detailed
+  </MenuItem>,
+  <MenuItem
+    onClick={() => {
+      MessageManager.clearMessages();
+    }}
+  >
+    Clear
+  </MenuItem>,
+];
+
+const DynamicDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+  });
+
+  return (
+    <div>
+      <Story />
+      <DropdownButton menuItems={menuItems} styleType="borderless">
+        Add Messages
+      </DropdownButton>
+    </div>
+  );
+};
+
+export const Dynamic: Story = {
```

Comment:
Open dynamic story: https://itwin.github.io/appui/746/?path=/story/components-messagecenterfield--dynamic

Steps:
- Click `Add message` -> `Info` (green bubble is rendered as excepted)
- Click `Add message` -> `Error` (red bubble is rendered as expected)
- Click on `Messages` field to open the popover (no bubble is rendered as expected)
- Click `Add message` -> `Info` (OUCH! red bubble is rendered, but green is expected)